src/components/WalletAssetDetails.vue

```diff
@@ -287,10 +287,12 @@ export default class WalletAssetDetails extends Mixins(FormattedAmountMixin, Cop
     return asset && asset.symbol === KnownSymbols.XOR;
   }
 
-  get isCleanHistoryDisabled(): boolean {
```

Comment:
please keep it as disabled (without negotiations in template) if you don't need enabled getter in terms of this component